ui/mirage/factories/variable.js

```diff
@@ -0,0 +1,21 @@
+import { Factory } from 'ember-cli-mirage';
+import faker from 'nomad-ui/mirage/faker';
+
+export default Factory.extend({
+  id: () => faker.random.words(3).split(' ').join('/').toLowerCase(),
+  path() {
+    return this.id;
+  },
+  namespace: 'default',
+  items() {
+    return (
+      this.Items || {
+        [faker.database.column()]: faker.database.collation(),
+        [faker.database.column()]: faker.database.collation(),
+        [faker.database.column()]: faker.database.collation(),
+        [faker.database.column()]: faker.database.collation(),
+        [faker.database.column()]: faker.database.collation(),
```

Comment:
`question` `non-blocking`: Should we wrap this into a utility to create a mock object on the fly?

`question`: The property name is `items` which infers a list, but we're generating a single object from faker in the second part here.

Reply:
`items` is the desired API convention here, and I think it's warranted even though it's an object not an array. Another fine name might have been "entries", but either way: it's a key/value object